packages/wallets/src/features/cfd/screens/CFDSuccess/CFDSuccess.scss

```
@@ -25,6 +25,10 @@
         height: 2.5rem;
         width: 5rem;
 
+        @include rtl {
+            border-radius: 1rem 0;
+        }
+
```

```suggestion
        @include rtl {
            border-radius: 1rem 0;
        }

```
Could we replace it with `border-start-end-radius` ?

---

Updated to border-start-end-radius and border-end-start-radius 👍